randomizer/text/character_formatter.py

```diff
@@ -0,0 +1,7 @@
+from enum import IntEnum
+
+class NpcNextCharacterFormatter(IntEnum):
```

Comment:
This may have been missed in my commit message, but I was planning for the future. I wanted to allow for modifying NPC text hints, and this batch of values is what is needed for that. It may have been a little premature, though.

Reply:
Ah whoops, I did miss that.  Makes sense.